community/CONTRIBUTING.md

```diff
@@ -5,6 +5,192 @@ Thanks for considering contributing to OPEA project. The contribution process is
 
 ## All The Ways To Contribute
 
+### Contribute a GenAIComponent
```

Comment:
should have a blank here? "Contribute a GenAI Component"?

Reply:
added